src/state/features/userSlice.ts

```diff
@@ -18,11 +18,11 @@ export const userSlice = createSlice({
   name: 'userInfo',
   initialState,
   reducers: {
-    login: (state: userInfoType, action: PayloadAction<userInfoType>) => {
+    login: (state: userInfoType, action: PayloadAction<any>) => {
```

Comment:
We should avoid using any if possible. From the code the type seems to be 

```typescript
interface LoginPayload {
  name: string;
  email: string;
  imageUrl: string;
}
```

Reply:
Maybe the payload will be a very complicated object type value. (ex. {cid: string, name: string, email: string, userInfo: any, etc.}) Thus, I consider to change the userSlice data.